site/en/docs/android/custom-tabs/guide-engagement-signals/index.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: "layouts/doc-post.njk"
+title: Measuring user engagement
+seoTitle: "Guide: measuring user engagement in Android Custom Tabs"
```

Comment:
FYI. `seoTitle` isn't required. It's for disambiguating pages with the same title. For example, we have gobs of pages simply titled "Overview".

Reply:
:-) I've added that feature. For SEO I think it's helpful to make the context more explicit.

Reply:
I would suggest then that what you actually need is more descriptive titles and descriptions. That would serve purposes beyond SEO.